app/controllers/candidate_interface/after_sign_in_controller.rb

```diff
@@ -29,15 +30,27 @@ def redirect_to_path_if_path_params_are_present
       redirect_to params[:path] if params[:path].present?
     end
 
+    def redirect_to_prefill_if_sandbox_user_has_blank_application
+      if HostingEnvironment.sandbox_mode? && current_application.blank_application?
+        course = course_from_find
+
+        if course
+          key = "prefill_application_#{current_user.id}"
+          prefill_application_store = WizardStateStores::RedisStore.new(key: key)
+          prefill_application_store.write({ prefill: true, course_id: course.id }.to_json)
```

Comment:
Do we need `prefill: true` — isn't the presence of the key in the cache enough?

Reply:
(I do agree the hash is a good idea)

Reply:
Yep true - the `course_id` should suffice 👍